tests/errmsgs/twrongcolon.nim

```diff
@@ -1,10 +1,8 @@
 discard """
-errormsg: "in expression ' do:"
+errormsg: "in expression '((890))'"
 nimout: '''
-twrongcolon.nim(10, 12) Error: in expression ' do:
-  890': identifier expected, but found ''
+twrongcolon.nim(8, 12) Error: in expression '((890))': identifier expected, but found ''
 '''
-
 """
 
 var n: int : 890
```

Comment:
I don't think this is a good test. This actually parses as:

```
VarSection
  IdentDefs
    Ident "n"
    Ident "int"
    Call
      Empty
      StmtList
        IntLit 890
```

because it's parsed as `identColonEquals colonBody?`. Either this weird parse should have a purpose and a specific error message should be given like "colon used instead of equals for variable assignment" or the grammar should be changed so `colonBody` is only parsed after an `=` and the error becomes "unexpected token: ':'"